app/Http/Controllers/Service/RegistrationController.php

```diff
@@ -280,7 +281,39 @@ function ($child) {
         );
 
         // Fetch Registration and Family
-        $registration = Registration::where('id', $request->get('registration'))->first();
+        $registration = Registration::findOrFail($request->get('registration'));
+
+
+        // Expect only filled fm variables
+        $fm = array_filter(
+            $request->only('fm_chart', 'fm_diary'),
+            function ($value) {
+                // Remove any null or empty responses;
+                return (isset($value) || ($value !== ''));
```

Comment:
Does that mean if a user deletes something - the value will remain as it was?

Reply:
So I am guessing these are checkboxes you are looking at and if they are ticked - you want the time they were saved recorded.

Hard to tell out of context - but I think that makes sense.

Reply:
They're chekboxes. 

a checked box returns it's value "1".

an uncheked checkbox is NOT submitted. Hence a hidden input of the same name with a value of '0' is drawn just before it. This gives us a "true/false" equiv on the box IF it's drawn.